app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseDialog.kt

```diff
@@ -0,0 +1,129 @@
+package org.simple.clinic.navigation.v2.fragments
```

Comment:
Can we rename the commit message to `Add BaseDialog as a base class for all dialogs supporting `